### PR TITLE
Arthur blaze/macro flask

### PR DESCRIPTION
**Pull request by ajlacey**
Checkout the [Engineer Workflow Doc](https://docs.google.com/presentation/d/1pdzuHG-yO7_qW6jiByaGTLztKZ6b72lBNqSdNT5WZQ8/edit) to make sure you're contributing correctly! :)

**[Tickets](https://trello.com/b/dYOQKJGv/reclass)** 
Trello Ticket: https://trello.com/c/ezkUqIgm
Additional Notes:

**[Dependencies](https://docs.google.com/document/d/1CM6OA2nlhI3DHnHdDEgySukx_tZ_wsG-C2YgraZRvJA/edit#heading=h.czpk4cbi7z4g)**

**[Explain Impact on DB](https://docs.google.com/document/d/1CM6OA2nlhI3DHnHdDEgySukx_tZ_wsG-C2YgraZRvJA/edit#heading=h.285lgcs4z84o)**

**[Testing](https://docs.google.com/document/d/1CM6OA2nlhI3DHnHdDEgySukx_tZ_wsG-C2YgraZRvJA/edit#heading=h.rs692kr58cf8)**

How did you test this? List of actions & expected behavior with helpful screenshots for reviewer: 
Utilized automated pytests to verify that macro flask could 'spin up' mini flask and get expected results for given inputs.

If this change is performance related, please document what tests were / should be done to verify improvement:

### CODE REVIEW

**Review by scottjr632 — DISMISSED** [review]
This is super cool!

The big thing is that in order for me to get this to work with docker I had to mount the docker socket to the container. Also I had to create a `docker-entrypoint.sh` file and update the `Dockerfile` accordingly. 

This relates to Issue #24 

Docker entrypoint looked like this
```bash
# docker-entrypoint.sh
#!/bin/sh

flask init-db
flask run --host 0.0.0.
```

And updated Dockerfile
```
# Removed the CMD
COPY ./docker-entrypoint.sh /app/docker-entrypoint.sh
RUN chmod +x /app/docker-entrypoint.sh

ENTRYPOINT [ "/app/docker-entrypoint.sh" ]
```

As well, I couldn't get this to run on my local machine for some reason. I suspect its due to how Docker Desktop works on Mac. This would be worth noting in the README as well. We can look at doing development inside docker containers in the future for this in case we can't figure out how to get this to work on Macs for local development.

**Comment by blazebissar** [comment]
Current state should address #24; but now the CI breaks when building mininet. Related PR attempts to fix that: [here](https://github.com/Project-Reclass/toynet-mininet/pull/8).

File: requirements.txt [hunk 1]
```diff
@@ -2,13 +2,17 @@ aniso8601==8.1.0
 apispec==4.4.1
```
[the remark this answers is not in the document]
Comment (blazebissar) [reply requirements.txt]: Thanks, added that in.

File: README.md [hunk 2]
```diff
@@ -33,6 +33,14 @@ $ deactivate
 
 More information [here](https://docs.python.org/3/library/venv.html)
```
[the remark this answers is not in the document]
Comment (blazebissar) [reply README.md:34]: This should work now, let me know if it's still broken.

**Review by takakonishimura — APPROVED** [review]
🚀 🚀 🚀 🚀 🚀

**Comment by takakonishimura** [comment]
@ajlacey this is ready to merge!!! 🔥 🥇 🔥

**Pull request merged**